Directory.Build.props

```diff
@@ -223,9 +225,36 @@
     <BuildingUAPAOTVertical Condition="'$(BuildingUAPAOTVertical)' == '' AND ('$(_bc_TargetGroup)'=='uapaot' OR '$(BuildAllConfigurations)' == 'true')">true</BuildingUAPAOTVertical>
   </PropertyGroup>
 
+  <!--
+    Import the arcade sdk with these requirements.
+    After:
+      Configuration setup
+
+    Before:
+      DebugType needs to be not be set to embedded at least for facades
+      OutputPath setup
+  -->
+  <PropertyGroup>
+    <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <_IsTestProjectBefore>$(IsTestProject)</_IsTestProjectBefore>
+    <IsPerformanceTestProject Condition="'$(IsPerformanceTestProject)' == '' AND '$(IsTestProject)' == 'true' AND $(MSBuildProjectName.EndsWith('Performance.Tests'))">true</IsPerformanceTestProject>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+    <!-- Need to reset our definition of IsTestProject after arcade sdk import -->
```

Comment:
Is this still required?  I thought we were going to agree upon a convention, or allow to turn off arcade's convention.

Reply:
Yes this is still required. @ViktorHofer will work on cleaning this up with his testing changes.